lib/Yam/Agama/Pom/RebootPage.pm

```diff
@@ -35,8 +36,17 @@ sub expect_is_shown {
 
 sub reboot {
     my ($self) = @_;
-    select_console('installation');
-    assert_and_click($self->{tag_reboot_button});
+    if (is_vmware) {
+        if (current_console() =~ /root/) {
+            enter_cmd 'reboot';
+        } else {
+            select_console('root-console');
+            enter_cmd 'reboot';
```

Comment:
two same lines `enter_cmd 'reboot';`.

Reply:
Code inside Page Objects should not have conditions, we use different design. I kindly ask for revert this PR in your slack channel.